src/main/java/io/github/mivek/parser/AbstractParser.java

```diff
@@ -78,7 +66,7 @@ protected void parseMinimalVisibility(final Visibility pVisibility, final String
      * @return a weather condition with its intensity, its descriptor and the
      * phenomenon.
      */
-    protected WeatherCondition parseWeatherCondition(final String weatherPart) {
+    WeatherCondition parseWeatherCondition(final String weatherPart) {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/mivek/MetarParser/pullRequest?prid=4593741)